docs/examples/simple_goto.rst

```diff
@@ -19,65 +19,83 @@ to use more appropriate positions for your own vehicle.
     * *Rover* will ignore the ``takeoff`` command and will then stick at the altitude check.
    
 
+.. figure:: simple_goto_example_copter_path.png
+   :width: 50 %
+   :alt: Setting destination using position and changing speed and ROI
+
+   Simple Goto Example: Flight path
+   
+   
 
 Running the example
 ===================
 
-The vehicle and DroneKit should be set up as described in :ref:`get-started`.
+The example can be run as described in :doc:`running_examples` (which in turn assumes that the vehicle
+and DroneKit have been set up as described in :ref:`get-started`).
 
 If you're using a simulated vehicle, remember to :ref:`disable arming checks <disable-arming-checks>` so 
 that the example can run.
 
-Once MAVProxy is running and the API is loaded, you can start the example by typing: ``api start simple_goto.py``.
+In summary, after cloning the repository:
 
-.. note:: 
+#. Navigate to the example folder as shown:
+
+   .. code-block:: bash
+
+       cd dronekit-python\examples\simple_goto\
 
-    The command above assumes you started the *MAVProxy* prompt in a directory containing the example script. If not, 
-    you will have to specify the full path to the script (something like):
-    ``api start /home/user/git/dronekit-python/examples/simple_goto/simple_goto.py``.
 
-.. tip::	
+#. Start the example, passing the :ref:`connection string <get_started_connect_string>` you wish to use in the ``--connect`` parameter:
+
+   .. code-block:: bash
+
+       python simple_goto.py --connect 127.0.0.1:14550
+
+   .. note::
+   
+       The command parameter above is the default, and may be omitted. This
+       connects to SITL on udp port 127.0.0.1:14550.
+
+       
+.. tip::
 
     It is more interesting to watch the example above on a map than the console. The topic :ref:`viewing_uav_on_map` 
     explains how to set up *Mission Planner* to view a vehicle running on the simulator (SITL).
 
-On the *MAVProxy* console you should see (something like):
+On the command prompt you should see (something like):
 
-.. code-block:: python
+.. code-block:: bash
 
-    MAV> api start simple_goto.py
-    STABILIZE> Basic pre-arm checks
+    \dronekit-python\examples\simple_goto>simple_goto.py
+    Connecting to vehicle on: 127.0.0.1:14550
+    >>> ☺APM:Copter V3.4-dev (e0810c2e)
```

Comment:
Thanks. Done. Note, for a bunch of these examples I'll have to regenerate this output once blocking bugs are fixed.